src/main/java/frc/robot/utilities/EncoderConversions.java

```diff
@@ -42,6 +52,16 @@ public static double RPMToFalcon(double RPM, double gearRatio) {
         return sensorCounts;
     }
 
+    /**
+     * Default gear ratio of 1
+     *
+     * @param RPM RPM of mechanism
+     * @return RPM of Mechanism
+     */
+    public static double RPMToFalcon(double RPM) {
```

Comment:
no default functions here

Reply:
it is not my case, should I still change it?